data/hai/hai reveal 1 intro.txt

```diff
@@ -11,48 +11,776 @@
+				decline
+
+
+
+mission "HR: Call for Aid with Sandsnap"
+	name "Join an effort on <planet>"
+	job
+	repeat
+	description "Experienced combat pilots are requested to apply at the spaceport on <destination> for a project to reactivate the Sandsnap station."
+	to offer
+		not "hr: sandsnap project"
+		not "Hai Reveal Concern Period: Reactivating Sandsnap: declined"
+	source
+		government "Hai"
+		not planet "Makerplace"
+	destination "Makerplace"
+	to complete
+		has "hr: sandsnap project"
+	on complete
+		payment 1000 150
+		dialog `Having accepted the job, the initial retainer of <payment> appears in your account.`
+
+
+
+mission "Hai Reveal Concern Period: Reactivating Sandsnap"
+	priority
+	name "Transport the Stationkeeper to <planet>"
+	description "Safely bring the Stationkeeper to <destination> to reactivate the station."
+	source "Makerplace"
+	destination "Sandsnap"
+	passengers 1
+	to offer
+		has "HR: Call for Aid with Sandsnap: accepted"
+	blocked `You are advised that participation in the Sandsnap Project will require at least 1 bunk free for passengers.`
+	on offer
+		conversation
+			`	Entering the spaceport requires one to walk around a very large holosign giving directions to the Sandsnap Project's office.`
+			choice
+				`	(Check out the project.)`
+				`	(Come back later.)`
+					defer
+				`	(Ignore it, you've changed your mind.)`
+					decline
+			`	There is a greeter, or perhaps a security guard, at the door who appraises you as you approach. "You must be here for the project," he says. "Identification please."`
+			`	After a quick check of your ID he gives you a very professional Hai smile. "Welcome to the Sandsnap Project. Your credentials are sufficient to apply so please, do come in. We hope you'll find the project interesting and the effort rewarding."`
+			`	As you enter a harried Hai gestures you over with enthusiasm. "Greetings," he checks his tablet. "Captain <last>! Your presence in these dire times is greatly appreciated - it has been at least a hundred years since we have had such a threat to our way of life! You must have noticed the spike in pirate activity. Perhaps our northern brethren have had a point all these years."`
+			`	He stops himself. "Not that our politics probably concern you. Let us cut to the matter at hand, shall we?"`
+			choice
+				`	(Nod.)`
+					goto border
+				`	"What are you talking about?"`
+				`	"Not at all, I love politics."`
+			`	He seems surprised, "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
```

Comment:
```suggestion
			`	He seems surprised. "Oh. Well in that case, the Unfettered have been militarizing a lot in the last 20 years, forcing us to do the same in order to protect merchants from their raids. They may be right that we have grown weak, if the success of those pirate raids is any indication; security forces were simply not ready for them. We here are in the business of ensuring this will not happen again. Mark my words, nothing like an arms manufacturer setting up a support base to really wake a few people up.`
```
I've got déjà vu about this suggestion.